### DISCUSSION TITLE
Add loras inline support to wildcards

### DISCUSSION THREAD

**Discussion by webestilo** [lllyasviel/Fooocus#3697] [Ideas]
I would like to request the addition of inline LoRA support to the wildcards system. Specifically, I propose the functionality to use LoRAs with the following inline format:

keyword1, keyword2, <LORA_NAME:WEIGHT>

This feature would enhance the flexibility and customization within the wildcards system by allowing users to directly apply LoRAs with specific weights in their prompts. This would streamline workflows, particularly in complex generation scenarios that require fine-tuning with multiple LoRAs.

**Comment by mashb1t** [comment]
this already works, see https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#inline-loras
